### PR TITLE
Release 0.2.0 changelog & version bump

### PR DESCRIPTION
**Pull request by arnoldsandoval-okta**

### CODE REVIEW

File: packages/odyssey/CHANGELOG.md [hunk 1]
```diff
@@ -9,6 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Rebrand type and color changes
 
+## [0.2.0] - 2020-04-30
+
+### Added
+- Status: New component and documentation
+- Tab: New component and documentation
+- Tag: New component and documentation
+- Toast: New component and documentation
```
Comment (edburyenegren-okta) [inline packages/odyssey/CHANGELOG.md:18]: Toast isn't making it!

**Comment by edburyenegren-okta** [comment]
Lookin' good. Do we want to make any formatting changes to our line items based on conventional commit standards? Or resolve that a bit later?

File: packages/odyssey/CHANGELOG.md (hunk 1 above)
Comment (arnoldsandoval-okta) [reply packages/odyssey/CHANGELOG.md:18]: oh man, good catch, removing

**Comment by arnoldsandoval-okta** [comment]
@edburyenegren-okta I haven't thought about it much honestly. If you have any preference/guidelines you want us to stick to im happy to implement.

**Comment by edburyenegren-okta** [comment]
@arnoldsandoval-okta let's tackle separately instead of trying to find a happy solution before tomorrow. these notes have the clarity they need

**Pull request closed without merging**